apps/web/src/components/DepositAndBorrow/Deposit/AssetsToDepositTable.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { FunctionComponent } from 'react'
+import {
+  Button,
+  CellProps,
+  ColumnLayout,
+  Icon,
+  StackLayout,
+  Table,
+  Text,
+} from '@bloxifi/ui'
+
+import { WalletContainer } from '@/containers/WalletContainer'
+
+export const AssetsToDepositTable: FunctionComponent = () => {
+  const {
+    state: { reserves },
+  } = WalletContainer.useContainer()
+
+  const columns = {
+    assets: {
+      header: 'Assets',
+      Cell: ({ data: { symbol, icon, fullName } }: any) => (
```

Comment:
@aleksaToljic  Here I got an error for Icon component because it doesn't accept name as "unknown" that we have in CellProps. (Type 'unknown' is not assignable to type 'string') Not sure how we can fix this